README.md

```diff
@@ -83,6 +82,43 @@ All components receive configuration via environment variables. The following va
 | `CASTELLUM_AUDIT_SILENT`<br>(API only) | `false` | Disable audit event logging to standard output. |
 | `OS_...` | *(required)* | A full set of OpenStack auth environment variables for Castellum's service user. See [documentation for openstackclient][os-env] for details. |
 
+All components also expect a positional argument containing the path of a YAML configuration file.
+Below is a working example for a configuration file:
+
+```yaml
+max_asset_sizes:
+  - asset_type: 'nfs-shares(-group:.+)?'
+    value: 16384
+
+project_seeds:
+  - project_name: myproject
+    domain_name: mydomain
+    resources:
+      nfs-shares:
+        critical_threshold: { usage_percent: 95 }
+        size_steps: { percent: 10 }
+        size_constraints: { max_size: 8192 }
+    disabled_resources:
+      - 'project-quota:.*'
+```
+
+The following fields are allowed:
+
+| Field | Type | Explanation |
+| ----- | ---- | ----------- |
+| `max_asset_sizes` | array of objects | If present, resource configurations for matching asset types will only be allowed if they include a compatible `max_size` constraint. |
+| `max_asset_sizes[].asset_type` | list of regexes | Regex that specifies which asset types this constraint applies to. |
+| `max_asset_sizes[].value` | integer | Highest permissible value for the `max_size` constraint on matching resources. |
+| `project_seeds` | array of objects | Specification of projects that will have resources configured. The observer will apply these seeds, and the API will reject attempts to manually override the seeded configuration. |
+| `project_seeds[].project_name` | string | Name (not ID!) of the project. |
+| `project_seeds[].domain_name` | string | Name (not ID!) of the domain containing the project. |
+| `project_seeds[].resources.$type` | object | Specification of a resource that will be statically configured in this project. The contents of this object must be identical to the payload that will be accepted for `PUT /v1/projects/$project_id/resources/$type`. See [API spec](./api-spec.md) for details. |
+| `project_seeds[].disabled_resources` | list of strings | A list of regexes. Any asset type that matches one of these regexes will have autoscaling disabled and forbidden in this project. This can be used to delete resources that were configured by an earlier version of the seed. |
+
+All regexes are matched against the entire asset type string, i.e. a leading `^` and trailing `$` are always added implicitly.
+
+When applying project seeds, projects that do not exist in Keystone will be skipped without logging an error.
```

Comment:
I have the feeling that this will silently ignore typos.

Reply:
True, but typos are reasonably unlikely since we don't often edit the seed file (and especially not the project/domain names). The value of this lenient behavior is that we can share the same seed between all deployments, even if some projects do not exist in all regions.

Reply:
if you think this won't be an issue, then fine for me

Reply:
I have added a log line for when projects were missing during the seeding run.